ert_gui/model/snapshot.py

```diff
@@ -271,6 +279,10 @@ def _job_data(self, index, node, role):
             elif index.column() == 5:
                 data = node.data.get("data")
                 return data.get("max_memory_usage") if data else QVariant()
+            elif index.column() == 6:
+                return node.data.get("stdout")
+            elif index.column() == 7:
+                return node.data.get("stderr")
```

Comment:
Seems to map to `"OPEN"` per the current implementation.

Reply:
Would it be possible to alter the presentation in the JobListProxyModel? Or is it best to do here?

Reply:
Yeah, it's possible… Maybe this can be done in the view however?